src/fn/sod323.ts

```diff
@@ -0,0 +1,99 @@
+import type { AnySoupElement, PcbSilkscreenPath } from "circuit-json"
+import { z } from "zod"
+import { rectpad } from "../helpers/rectpad"
+import { silkscreenRef, type SilkscreenRef } from "src/helpers/silkscreenRef"
+import { length } from "circuit-json"
+
+export const sod_def = z.object({
+  fn: z.string(),
+  num_pins: z.literal(2).default(2),
+  w: z.string().default("3.30mm"),
```

Comment:
@techmannih From where are you getting this `3.30` value? Cause I can't see in the kicad viewer any width being `3.30`!!